client/src/Pages/Home.jsx

```diff
@@ -23,27 +25,108 @@ class Home extends React.Component{
+                "person": "Geel Hoofd",
+                "messageType": "3"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "3"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "3"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "3"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "3"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "2"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "1"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "0"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "0"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "0"
+            },
+            {
+                "id": "40,",
+                "person": "Geel Hoofd",
+                "messageType": "2"
+            },
+        ];
         return(
-        <div id="intro">
-            <form>
-            <table>
-                <tbody>
-                <tr>
-                    <td><input type="text" id="uname" name="uname" placeholder="Gebruikersnaam" value={this.state.uname} onChange={this.handleInputChange} /></td>
-                </tr>
-                <tr>
-                    <td><input type="password" id="pass" name="pass" placeholder="Wachtwoord" value={this.state.pass} onChange={this.handleInputChange} /></td>
-                </tr>
-                <tr>
-                    <td><a href="#">Wachtwoord vergeten?</a></td>
-                </tr>
-                <tr>
-                    <button>Inloggen</button>
-                </tr>
-                </tbody>
-            </table>
-            </form>
-        </div>
+            <div className="Home">
+                <div align='center' className="LinebreakPrevent">
+                    <Link to='./Rooster'>
+                        <figure>
+                            <img src='https://svgshare.com/i/FKH.svg' alt="Calendar Icon" width='150'/>
+                            <figcaption>ROOSTER</figcaption>
+                        </figure>
+                    </Link>
+                    <Link to='./Salaris'>
+                        <figure>
+                            <img src='https://svgshare.com/i/FL7.svg' alt="money" width='150'/>
+                            <figcaption>SALARIS</figcaption>
+                        </figure>
```

Comment:
Kijk eens hiernaar voor svg's https://medium.com/@rossbulat/working-with-svgs-in-react-d09d1602a219